community/goobook/APKBUILD

```diff
@@ -0,0 +1,29 @@
+# Contributor: Ivan Tham <pickfire@riseup.net>
+# Maintainer: Ivan Tham <pickfire@riseup.net>
+pkgname=goobook
+pkgver=3.3
+pkgrel=0
+pkgdesc="Access Google contacts from the command line"
+url="https://gitlab.com/goobook/goobook"
+arch="noarch"
+license="GPL-3.0"
```

Comment:
GPL-3.0 is not a valid spdx identifier

Reply:
@maxice8 thank you! I have fixed this. It was sneaking through the sanity check so I have also filed https://github.com/alpinelinux/abuild/pull/84